web-tier/src/test/resources/test-data/ListOfCategoriesWithDishes.json

```diff
@@ -0,0 +1 @@
+[{"id":12345,"name":"First category","dishes":[{"id":22222,"calorific":1.12,"proteins":1.12,"fats":1.12,"carbs":1.12,"weight":20.0,"name":"dishView1","categoryId":12345,"products":[{"id":101010,"calorific":1.12,"proteins":1.12,"fats":1.12,"carbs":1.12,"weight":20.0,"name":"first prod","categoryId":77777},{"id":121212,"calorific":1.12,"proteins":1.12,"fats":1.12,"carbs":1.12,"weight":20.0,"name":"second prod","categoryId":77777}]},{"id":33333,"calorific":1.12,"proteins":1.12,"fats":1.12,"carbs":1.12,"weight":20.0,"name":"dishView2","categoryId":12345,"products":[]}]},{"id":33321,"name":"Empty","dishes":[{"id":44444,"calorific":2.0,"proteins":1.12,"fats":1.12,"carbs":1.12,"weight":20.0,"name":"dishView3","categoryId":33321,"products":[]}]}]
```

Comment:
Please make it "human readable", using "Code\Reformat Code" menu, or Ctrl|+Alt+L
and move it to `dish` folder